### PR TITLE
Penalty keeper CenterJump

### LINKED ISSUE #997: Penalty Shootout Keeper
Our Keeper jumps directly after going down, likely because the ball gets velocity due to the sensor lag.
Ideas:
~- Just go down earlier (anyways good idea)~
~- ball only gets velocity in Play~
~- use whistle detection?~
- [ ] Ask the Devils if they would provide their Keeper motion since it looks soooo much better than ours
- [x] Implement the wide stance from the wide stance into the current keeper behavior

### PR DESCRIPTION
**Pull request by larskna**
## Why? What?

We added the WideStance to the penalty keeper für Center balls for to win better

Fixes #997 

## Ideas for Next Iterations (Not This PR)

Squat Motion improvments

## How to Test

Shot the Ball in the Penalty in the Center of the Goal

### CODE REVIEW

File: crates/control/src/penalty_shot_direction_estimation.rs [hunk 1]
```diff
@@ -25,8 +26,8 @@ pub struct CreationContext {}
 #[context]
 pub struct CycleContext {
     field_dimensions: Parameter<FieldDimensions, "field_dimensions">,
-    moving_distance_threshold:
-        Parameter<f32, "penalty_shot_direction_estimation.moving_distance_threshold">,
+    penalty_shot_paramters:
```
Comment (oleflb) [inline crates/control/src/penalty_shot_direction_estimation.rs:29]: typo

File: etc/parameters/default.json [hunk 2]
```diff
@@ -1042,7 +1042,9 @@
     }
   },
   "penalty_shot_direction_estimation": {
-    "moving_distance_threshold": 0.2
+    "moving_distance_threshold": 0.2,
+    "minimum_velocity": -0.5,
```
Comment (oleflb) [inline etc/parameters/default.json:1046]: This disables the velocity check, right?

**Review by oleflb — CHANGES_REQUESTED** [review]
Can't test, just code review

File: etc/parameters/default.json (hunk 2 above)
Comment (larskna) [reply etc/parameters/default.json:1046]: We dont have a velocity check in the main right now, this creates one

File: crates/control/src/penalty_shot_direction_estimation.rs (hunk 1 above)
Comment (larskna) [reply crates/control/src/penalty_shot_direction_estimation.rs]: fixed

**Review by oleflb — APPROVED** [review]

**Pull request merged**